Tasks/google-play-release/GooglePlay.ts

```diff
@@ -116,10 +116,10 @@ async function run() {
                 tl.debug(`Uploading APK ${apkFile}`);
                 const apk: googleutil.Apk = await googleutil.addApk(edits, packageName, apkFile);
                 tl.debug(`Uploaded ${apkFile} with the version code ${apk.versionCode}`);
-                if ((shouldPickObbForApk(apkFile, mainApkFile, shouldPickObbFile, shouldPickObbFileForAdditonalApks)) && (getObbFromParentDirectory(apkFile) !== null)) {
-                    const obb: googleutil.ObbResponse = await googleutil.addObb(edits, packageName, getObbFromParentDirectory(apkFile), apk.versionCode, 'main');
+                if ((shouldPickObbForApk(apkFile, mainApkFile, shouldPickObbFile, shouldPickObbFileForAdditonalApks)) && (getObbFromParentDirectory(apkFile, packageName, apk.versionCode) !== null)) {
```

Comment:
Could it make sense to pack all input variables for the `getObbFromParentDirectory` function into one object? It will prevent errors with mixed up input data.

Reply:
This point was actually raised during the last PR when having a chat with Lucas I guess. Readability would be decreased if we use object here. Should I still go ahead and use object here?

Reply:
I discuss this question with @anatolybolshakov and we decided that in this case, it's not necessary.  But could you please update commentary for `getObbFromParentDirectory` function with information about new arguments using [JSDoc](https://jsdoc.app/).

Reply:
Updated the comments